src/enrichment.cc

```diff
@@ -220,8 +220,10 @@ std::set<cyclus::BidPortfolio<cyclus::Material>::Ptr> Enrichment::GetMatlBids(
       Material::Ptr mat = req->target();
       double request_enrich = cyclus::toolkit::UraniumAssay(mat) ;
 
-      if (ValidReq(req->target()) && (request_enrich <= max_enrich)) {
-        Material::Ptr offer = Offer_(req->target());
+       if (ValidReq(req->target()) &&
+	  ((request_enrich < max_enrich) ||
+	   (cyclus::AlmostEq(request_enrich, max_enrich)))){
+         Material::Ptr offer = Offer_(req->target());
         commod_port->AddBid(req, offer, this);
       }
     }
```

Comment:
@rwcarlsen : As far as I see, your suggested loop above is identical to the PR I submitted?